eps/README.md

```diff
@@ -0,0 +1,106 @@
+
+### 4. Document Implementation Details
+
+Once the goals are set, begin filling out the implementation details. The details should include the following:
+
+- APIs and customer driven interfaces -- this includes CLI arguments, public libraries and SDKs, API resources and
+  objects.
+- Use cases.
+- Security considerations.
+- User experience and user interface changes.
+- Testing plans and considerations.
+- References.
+- Any alternatives considered and why we decided against them.
+
+Once the EP is merged, mark it as "Implementable".
+
+### 5. Prototype (optional)
+
+In some cases, you may be asked or want to create a prototype of the design. Prototypes are a good way to get early
+feedback about the implementation without investing too much time and effort on the small details. When creating a
+prototype of an EP, mark the EP as "Prototyping" and open a Draft PR. If the prototype is approved, you can move on to
+the next stage. Otherwise, you may be asked to make changes to the design and resubmit the EP as "Provisional".
+
+Note that the prototype is not expected to be merged, but should be used as a way to move an EP from "Provisional" to "
+Implementable".
+
+### 6. Implement
+
+Once the EP is marked as "Implementable", you can start implementing the proposed changes. Once the changes have been
+merged, the EP issue should be closed and the EP should be marked as "Completed".
+
+## Status
+
+Each EP has a status field that defines its current state. Each transition will require a PR to update the EP.
+
+* **Provisional:** The goals described by this EP have consensus but implementation details have not been agreed to yet.
+* **Prototyping:** An extension of `Provisional` which can be opted in to in order to indicate that there are some
+  active practical tests and experiments going on which are intended to be a part of the development of this EP.
+* **Implementable:** The goals and implementation details described by this EP have consensus but have not been fully
+  implemented yet.
+* **Completed:** This EP has been implemented.
+
+Although less common, some EPs may end up in one of the following states:
+
+* **Deferred:** We do not currently have bandwidth to handle this EP, it may be revisited in the future.
+* **Rejected:** This proposal was considered but ultimately rejected.
+* **Replaced:** This proposal was considered but ultimately replaced by a newer proposal.
+* **Withdrawn:** This proposal was considered but ultimately withdrawn by the author.
+
+## Format
+
+EPs should match the format of this [template](template.md).
+
+## Out of scope
+
+The following is out of scope for EPs:
+
+* Bug fixes
+* Gateway API features. However, some larger Gateway API features may require EPs if they require significant changes to the
+  architecture of the code.
+* Small changes (validation, documentation, fixups). It is always possible that the reviewers will determine a "small"
```

Comment:
small changes are captured here.
just like bug fixes.
However, both of these things are a judgement call on the behalf of the maintainers who will be receiving and reviewing the PR.
It should be clear that this judgement call will be made at a specific place in the workflow.  Not simply that they may be exceptions.

Reply:
That's what this note in the enhancement template is for:
```
<!--
NOTE: depending on the scope of the enhancement, you may be asked to use the EP
process to document your work: https://github.com/nginxinc/nginx-kubernetes-gateway/blob/main/eps/README.md
-->
```

Reply:
I reworked this section to try and make that clear. Let me know what you think.